syntax = "proto3";

package ccset;

message ccset_data {
    repeated int64 offsets = 1;
    repeated int64 cc_h = 2;
    repeated int64 cc_v = 3;
    message darr {
        repeated int64 arr = 1;
    }
    repeated darr cc_h_tags = 4;
    repeated darr cc_v_tags = 5;
    double meastime = 6;
}
